server/game/gameSystems/PutIntoPlaySystem.ts

```diff
@@ -56,11 +56,10 @@ export class PutIntoPlaySystem<TContext extends AbilityContext = AbilityContext>
     public override canAffect(card: Card, context: TContext): boolean {
         const contextCopy = context.copy({ source: card });
         const player = this.getPutIntoPlayPlayer(contextCopy);
-        const location = card.location;
-
+        const playAbleLocations = [Location.Resource, Location.Deck, Location.Discard];
         if (!super.canAffect(card, context)) {
             return false;
-        } else if (!card.canBeInPlay() || card.isInPlay() || (card.facedown && card.location !== Location.Resource)) {
+        } else if (!card.canBeInPlay() || card.isInPlay() || (card.facedown && !playAbleLocations.includes(card.location))) {
```

Comment:
Maybe we just remove the facedown check completely since I think it's actually impossible for it to fail now? The only facedown locations are resource and deck and those now have exceptions

Reply:
removed the facedown check!

Reply:
Unfortunatelly by removing the check multiple tests failed like timely intervention so I'm assuming we need to keep it.